string name
uint8 frame_id
uint8 height_id
float64 height
geometry_msgs/Point position
mrs_msgs/Reference[] points
uint8 terminal_action

